capabilityquerier/pkg/capabilityquerier/capabilityquerier.go

```diff
@@ -98,35 +114,45 @@ func requestCapabilityStatement(ctx context.Context, fhirURL *url.URL, client *h
 		return err
```

Comment:
I think it might make sense for the wellknown endpoint call to use whatever mime type is currently saved in the message instead of just using the fhir3PlusJSONMIMEType, and if there isn't one saved to just use that one, since i would think it's more likely that whatever mime type worked for the metadata endpoint will also work for the wellknown one!

Reply:
In cases where both fhir3PlusJSONMIMEType and fhir2PlusJSONMIMEType are already stored in the message struct, should there a preference as to which is called? Or do both if the http status is not 200?

Reply:
for now (basing this off of lizzie's previous comment as well) i think i would just choose one and if it doesn't work then don't bother checking with the other